gbfs-validator/versions/schemas/v3.0-RC/system_information.json

```
@@ -0,0 +1,1222 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC/gbfs.md#system_informationjson",
+  "description":
+    "Details including system operator, system location, year implemented, URL, contact info, time zone.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
+      "type": "integer",
+      "minimum": 1450155600
+    },
+    "ttl": {
+      "description":
+        "Number of seconds before the data in the feed will be updated again (0 if the data should always be refreshed).",
+      "type": "integer",
+      "minimum": 0
+    },
+    "version": {
+      "description":
+        "GBFS version number to which the feed conforms, according to the versioning framework (added in v1.1).",
+      "type": "string",
+      "const": "3.0-RC"
+    },
+    "data": {
+      "description": "Response data in the form of name:value pairs.",
+      "type": "object",
+      "properties": {
+        "system_id": {
+          "description":
+            "Identifier for this vehicle share system. This should be globally unique (even between different systems).",
+          "type": "string"
+        },
+        "languages": {
+          "description":
+            "List of languages used in translated strings. Each element in the list must be of type Language.",
+          "type": "array",
+          "pattern": "^[a-z]{2,3}(-[A-Z]{2})?$"
```

Let's fix the incorrect spec for "languages" property like in https://github.com/MobilityData/gbfs-json-schema/pull/90. Thank you!

```suggestion
          "items": {
            "description": "IETF BCP 47 language code.",
            "type": "string",
            "pattern": "^[a-z]{2,3}(-[A-Z]{2})?$"
          }
```

---

I've updated the files from https://github.com/MobilityData/gbfs-validator/pull/104#pullrequestreview-1491739949 so this is now included.